### PR TITLE
Replace page.js with app-route

### PR DESCRIPTION
**Pull request by TimvdLippe**
Another step for #283 `page.js` has been replaced with `app-route`.

Note: this PR removes the hash-routing and rewrites some routes. We might want to take another good look and update the server accordingly.

### CODE REVIEW

File: src/lancie-content.html [hunk 1]
```diff
@@ -99,7 +105,7 @@
         <lancie-request-password-reset-success></lancie-request-password-reset-success>
       </div>
 
-      <div data-route="order-check">
+      <div data-route="ordersuccess">
```
Comment (mgajanssen) [inline src/lancie-content.html]: I think a better rename would be `order-success`, this is in line with the other pages regarding transferring/ordering.

File: src/lancie-layout/lancie-layout.html [hunk 2]
```diff
@@ -69,11 +69,11 @@
...
         <paper-menu selected="_selected">
-            <div><a href="{{baseUrl}}/"><paper-item on-tap="_closeDrawer">Home</paper-item></a></div>
-            <div><a href="{{baseUrl}}/tickets"><paper-item on-tap="_closeDrawer">Tickets</paper-item></a></div>
-            <div><a href="{{baseUrl}}/my-area"><paper-item on-tap="_closeDrawer">My Area</paper-item></a></div>
-            <div><a href="{{baseUrl}}/seatmap"><paper-item on-tap="_closeDrawer">Seatmap</paper-item></a></div>
-            <paper-item on-tap="toShop">Shop</paper-item>
+          <div><a href$="{{route.prefix}}/../"><paper-item on-tap="_closeDrawer">Home</paper-item></a></div>
+          <div><a href$="{{route.prefix}}/tickets"><paper-item on-tap="_closeDrawer">Tickets</paper-item></a></div>
+          <div><a href$="{{route.prefix}}/my-area"><paper-item on-tap="_closeDrawer">My Area</paper-item></a></div>
+          <div><a href$="{{route.prefix}}/seatmap"><paper-item on-tap="_closeDrawer">Seatmap</paper-item></a></div>
```
Comment (mgajanssen) [inline src/lancie-layout/lancie-layout.html]: All the `href` links in here can be replaced to just an identifier corresponding to the items in lancie-content `page` values.

**Comment by mgajanssen** [comment]
Okay, this should do the job quite nicely. Back to you Tim!

**Comment by TimvdLippe** [comment]
Removing `<a>` removes the ability to `ctrl + click` and is less nice for crawlers of search engines. We need to find a solution that makes use of `<a>` but is not crap :smile:

File: src/lancie-app.html [hunk 3]
```diff
@@ -10,15 +10,16 @@
         display: block;
       }
     </style>
-    <lancie-layout route="{{route}}">
+    <lancie-layout page="{{page}}">
       <div id="content">
-        <lancie-content route="{{route}}"></lancie-content>
+        <lancie-content page="{{page}}"></lancie-content>
```
Comment (TimvdLippe) [inline src/lancie-app.html]: Let's name it route.

**Comment by TimvdLippe** [comment]
Yes is working now, nice!

**Pull request merged**